scrapy/extensions/closespider.py

```diff
@@ -32,7 +33,7 @@ def __init__(self, crawler):
         if self.close_on.get('errorcount'):
             crawler.signals.connect(self.error_count, signal=signals.spider_error)
         if self.close_on.get('pagecount'):
-            crawler.signals.connect(self.page_count, signal=signals.response_received)
+            crawler.signals.connect(self.response_received, signal=signals.response_received)
```

Comment:
This is the callback for the `response_received` signal. In `BaseExtension` this method is aptly named `response_received`. So for consistency I renamed it and now it overrides the correct base class method.